packages/zoe/src/zoeService/internal-types.js

```diff
@@ -77,9 +79,10 @@
  * @property {WithdrawPayments} withdrawPayments
  * @property {InitInstanceAdmin} initInstanceAdmin
  * @property {DeleteInstanceAdmin} deleteInstanceAdmin
- * @property {CreateZCFVat} createZCFVat
  * @property {ZoeInstanceAdminMakeInvitation} makeInvitation
  * @property {Issuer} invitationIssuer
+ * @property {Object} root of a RootAndAdminNode
+ * @property {AdminNode} adminNode of a RootAndAdminNode
```

Comment:
These properties were already there. This PR just adds the missing declarations.